include/zephyr/net/dns_resolve.h

```diff
@@ -774,6 +786,19 @@ static inline int dns_resolve_service(struct dns_resolve_context *ctx,
  */
 struct dns_resolve_context *dns_resolve_get_default(void);
 
+#if defined(CONFIG_DNS_RESOLVER_PACKET_FORWARDING)
+/**
+ * @brief Installs the packet forwarding callback to the DNS resolving context.
+ *
+ * @details When this callback is installed, a received message from DNS server
+ * will be passed to the application that has requested this callback.
+ */
+static inline void dns_resolve_enable_packet_forwarding(dns_resolve_pkt_fw_cb_t cb)
+{
+dns_resolve_get_default()->pkt_fw_cb = cb;
```

Comment:
Indentation needs fixing. And moreover - I think DNS resolver context pointer (`struct dns_resolve_context *ctx`) should also be provided as a parameter to the function. If the application wants to configure the default context, it can call `dns_resolve_get_default()` to get the poiner on its own.